### PR TITLE
Completed Home Page features/functionalities with example data

### PR DESCRIPTION
**Pull request by kallycao**
- Implemented swipe functionality for dog cards
- Implemented image gallery functionality for dog cards
- Installed geolib to convert distance between two users to miles
- Implemented various style changes to match with wireframe
- Various comments were left in as markers to various axios requests I will be making to integrate front/back-end. They will be omitted once my module is fully integrated.

### CODE REVIEW

File: frontend/components/home/UserCard.jsx [hunk 1]
```diff
@@ -1,39 +1,185 @@
...
+          }),
+        ]).start(() => {
+          handleSwipe(swipeDirection);
+          omitCard();
+        });
+      }
+    },
+  });
+
+  const calcDistance = (currItem) => {
```
Comment (KateFeaster) [inline frontend/components/home/UserCard.jsx:91]: Do you want me to include distance in the `GET: /home` route? I already have to calculate it for filtering, so it wouldn't take any extra effort

File: frontend/components/home/exampleData/userData.js [hunk 2]
```diff
@@ -61,8 +61,8 @@ export const userData = [
     username: 'Cocopuff',
     dog_name: 'Coco',
     breed: 'Pomeranian',
-    size: 'small',
-    energy: 'high',
+    size: 'Small',
+    energy: 'High',
```
Comment (KateFeaster) [inline frontend/components/home/exampleData/userData.js:65]: Heads up that this data will be coming from the server all lowercase

**Review by KateFeaster — APPROVED** [review]
Looks great! One comment and one question, but nothing that should stop the merge. Great work!

File: frontend/components/home/UserCard.jsx (hunk 1 above)
Comment (kallycao) [reply frontend/components/home/UserCard.jsx:91]: That would be amazing Kate! Processing that on the backend should speed up things on the front-end significantly. I appreciate it, thanks so much.

File: frontend/components/home/exampleData/userData.js (hunk 2 above)
Comment (kallycao) [reply frontend/components/home/exampleData/userData.js:65]: Noted! Thank you for the heads up Kate. I'll make the appropriate styling changes on the front end.

**Review by boyiq — APPROVED** [review]
Looks great Kally! Good job on everything :)

**Pull request merged**